packages/flutter_bloc/lib/src/bloc_listener.dart

```diff
@@ -5,6 +5,7 @@ import 'package:flutter/widgets.dart';
 import 'package:bloc/bloc.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:provider/provider.dart';
+import 'package:nested/nested.dart';
```

Comment:
@rrousselGit can you please address https://github.com/rrousselGit/provider/pull/243/files#r359383148 so that this import can be removed?